### PR TITLE
feat: add logging_config and monitoring_config to autopilot modules

### PR DESCRIPTION
**Pull request by pocesar**
Matches the other submodules inputs

### CODE REVIEW

**Review by apeabody — CHANGES_REQUESTED** [review]
Thanks for the contribution @pocesar!

Please update the template as well: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/CONTRIBUTING.md#templating

**Comment by apeabody** [comment]
Hi @pocesar - Still something with the module generation from template:

```
Checking submodule's files generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/modules/beta-autopilot-private-cluster/cluster.tf /tmp/tmp.sFm4Tid78E/workspace/modules/beta-autopilot-private-cluster/cluster.tf
350,364d349
<   dynamic "logging_config" {
<     for_each = length(var.logging_enabled_components) > 0 ? [1] : []
< 
<     content {
<       enable_components = var.logging_enabled_components
<     }
<   }
< 
<   dynamic "monitoring_config" {
<     for_each = length(var.monitoring_enabled_components) > 0 ? [1] : []
<     content {
<       enable_components = var.monitoring_enabled_components
<     }
<   }
< 
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/modules/beta-autopilot-private-cluster/variables.tf /tmp/tmp.sFm4Tid78E/workspace/modules/beta-autopilot-private-cluster/variables.tf
327,337d326
< variable "monitoring_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
<   default     = []
< }
< 
< variable "logging_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
<   default     = []
< }
535a525,536
> }
> 
> variable "monitoring_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
>   default     = []
> }
> 
> variable "logging_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
>   default     = []
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/modules/beta-autopilot-public-cluster/cluster.tf /tmp/tmp.sFm4Tid78E/workspace/modules/beta-autopilot-public-cluster/cluster.tf
329,343d328
<   dynamic "logging_config" {
<     for_each = length(var.logging_enabled_components) > 0 ? [1] : []
< 
<     content {
<       enable_components = var.logging_enabled_components
<     }
<   }
< 
<   dynamic "monitoring_config" {
<     for_each = length(var.monitoring_enabled_components) > 0 ? [1] : []
<     content {
<       enable_components = var.monitoring_enabled_components
<     }
<   }
< 
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/modules/beta-autopilot-public-cluster/variables.tf /tmp/tmp.sFm4Tid78E/workspace/modules/beta-autopilot-public-cluster/variables.tf
291,301d290
< variable "monitoring_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
<   default     = []
< }
< 
< variable "logging_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
<   default     = []
< }
499a489,500
> }
> 
> variable "monitoring_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
>   default     = []
> }
> 
> variable "logging_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
>   default     = []
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/modules/beta-private-cluster/variables.tf /tmp/tmp.sFm4Tid78E/workspace/modules/beta-private-cluster/variables.tf
814a815,826
> variable "monitoring_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
>   default     = []
> }
> 
> variable "logging_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
>   default     = []
> }
> 
831,842d842
< }
< 
< variable "monitoring_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
<   default     = []
< }
< 
< variable "logging_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
<   default     = []
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/modules/beta-private-cluster-update-variant/variables.tf /tmp/tmp.sFm4Tid78E/workspace/modules/beta-private-cluster-update-variant/variables.tf
814a815,826
> variable "monitoring_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
>   default     = []
> }
> 
> variable "logging_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
>   default     = []
> }
> 
831,842d842
< }
< 
< variable "monitoring_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
<   default     = []
< }
< 
< variable "logging_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
<   default     = []
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/modules/beta-public-cluster/variables.tf /tmp/tmp.sFm4Tid78E/workspace/modules/beta-public-cluster/variables.tf
778a779,790
> variable "monitoring_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
>   default     = []
> }
> 
> variable "logging_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
>   default     = []
> }
> 
795,806d806
< }
< 
< variable "monitoring_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
<   default     = []
< }
< 
< variable "logging_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
<   default     = []
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/modules/beta-public-cluster-update-variant/variables.tf /tmp/tmp.sFm4Tid78E/workspace/modules/beta-public-cluster-update-variant/variables.tf
778a779,790
> variable "monitoring_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
>   default     = []
> }
> 
> variable "logging_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
>   default     = []
> }
> 
795,806d806
< }
< 
< variable "monitoring_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
<   default     = []
< }
< 
< variable "logging_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
<   default     = []
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/modules/private-cluster/variables.tf /tmp/tmp.sFm4Tid78E/workspace/modules/private-cluster/variables.tf
779a780,791
> variable "monitoring_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
>   default     = []
> }
> 
> variable "logging_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
>   default     = []
> }
> 
796,807d807
< }
< 
< variable "monitoring_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
<   default     = []
< }
< 
< variable "logging_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
<   default     = []
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/modules/private-cluster-update-variant/variables.tf /tmp/tmp.sFm4Tid78E/workspace/modules/private-cluster-update-variant/variables.tf
779a780,791
> variable "monitoring_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
>   default     = []
> }
> 
> variable "logging_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
>   default     = []
> }
> 
796,807d807
< }
< 
< variable "monitoring_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
<   default     = []
< }
< 
< variable "logging_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
<   default     = []
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/variables.tf /tmp/tmp.sFm4Tid78E/workspace/variables.tf
743a744,755
> variable "monitoring_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
>   default     = []
> }
> 
> variable "logging_enabled_components" {
>   type        = list(string)
>   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
>   default     = []
> }
> 
760,771d771
< }
< 
< variable "monitoring_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
<   default     = []
< }
< 
< variable "logging_enabled_components" {
<   type        = list(string)
<   description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
<   default     = []
Error: submodule's files generation has not been run, please run the
'make build' command and commit changes
```

**Comment by pocesar** [comment]
@apeabody had to shuffle the blocks around to be able to not increase the `if`/`else`/`endif` nesting on non-autopilot settings, created this big changeset

File: autogen/main/cluster.tf.tmpl [hunk 1]
```diff
@@ -94,42 +94,53 @@ resource "google_container_cluster" "primary" {
...
   }
 {% endif %}
-{% if autopilot_cluster != true %}
-  # only one of logging/monitoring_service or logging/monitoring_config can be specified
-  {% if beta_cluster %}
-  logging_service = local.cluster_telemetry_type_is_set || local.logmon_config_is_set ? null : var.logging_service
-  {% else %}
-  logging_service = local.logmon_config_is_set ? null : var.logging_service
-  {% endif %}
+
```
Comment (apeabody) [inline autogen/main/cluster.tf.tmpl:97]: nit: let's remove this line for spacing
```suggestion
```

File: autogen/main/cluster.tf.tmpl [hunk 2]
```diff
@@ -94,42 +94,53 @@ resource "google_container_cluster" "primary" {
...
   }
+
+{% if autopilot_cluster != true %}
+  # only one of logging/monitoring_service or logging/monitoring_config can be specified
+  {% if beta_cluster %}
+  logging_service = local.cluster_telemetry_type_is_set || local.logmon_config_is_set ? null : var.logging_service
+  {% else %}
+  logging_service = local.logmon_config_is_set ? null : var.logging_service
+  {% endif %}
+
```
Comment (apeabody) [inline autogen/main/cluster.tf.tmpl:137]: nit: I think we can drop this line as well
```suggestion
```

File: autogen/main/variables.tf.tmpl [hunk 3]
```diff
@@ -852,6 +852,18 @@ variable "timeouts" {
...
+variable "monitoring_enabled_components" {
+  type        = list(string)
+  description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
+  default     = []
+}
+
+variable "logging_enabled_components" {
+  type        = list(string)
+  description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, CONTROLLER_MANAGER, SCHEDULER, and WORKLOADS. Empty list is default GKE configuration."
+  default     = []
```
Comment (apeabody) [inline autogen/main/variables.tf.tmpl:864]: optional: might be nice to validate the list values: https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules

File: autogen/main/variables.tf.tmpl [hunk 4]
```diff
@@ -852,6 +852,18 @@ variable "timeouts" {
   }
 }
 
+variable "monitoring_enabled_components" {
+  type        = list(string)
+  description = "List of services to monitor: SYSTEM_COMPONENTS, APISERVER, SCHEDULER, CONTROLLER_MANAGER, STORAGE, HPA, POD, DAEMONSET, DEPLOYMENT, STATEFULSET, KUBELET, CADVISOR and DCGM. In beta provider, WORKLOADS is supported on top of those 12 values. (WORKLOADS is deprecated and removed in GKE 1.24.) KUBELET and CADVISOR are only supported in GKE 1.29.3-gke.1093000 and above. Empty list is default GKE configuration."
+  default     = []
```
Comment (apeabody) [inline autogen/main/variables.tf.tmpl:858]: optional: might be nice to validate the list values: https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules

**Review by apeabody — CHANGES_REQUESTED** [review]
Thanks for the contribution @pocesar!

A few very minor nits before merge.

File: autogen/main/variables.tf.tmpl (hunk 4 above)
Comment (pocesar) [reply autogen/main/variables.tf.tmpl:858]: done

Comment (apeabody) [reply autogen/main/variables.tf.tmpl:858]: Thanks!

File: autogen/main/variables.tf.tmpl (hunk 3 above)
Comment (apeabody) [reply autogen/main/variables.tf.tmpl:886]: Thanks!

**Review by apeabody — APPROVED** [review]

**Pull request merged**